app/controllers/api/comments_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Api::CommentsController < Api::ApplicationController
   before_action :set_mobile_user
-  before_action :update_username_if_changed
+  before_action :update_first_and_last_name, unless: :first_and_last_name_present?
```

Comment:
Trochę słabo, że taka logika jest w kontrolerze. Mamy takie opcje:
- przenieść to do `MobileUser`
- przenieść to do `CommentMaker`
- przenieść to do nowego serwisu, np. `MobileUserIdentityParams`

Pewnie ostatnia najlepsza, coś w tym stylu:
`if current_debate`
..`MobileUserIdentityParams.new(@mobile_user).update(first_name: params.dig(:first_name), last_name: params.dig(:last_name))`
..`...`
`end`

`class MobileUserIdentityParams`
..`...`
..`def update(first_name:, last_name:)`
....`mobile_user.update(first_name: first_name, last_name: last_name) unless has_identity?`
..`end`
..` `
..`private`
..` `
..`def has_identity?`
....`mobile_user.first_name.present? && mobile_user.last_name.present?`
..`end` 
`end`

Co myślisz? Może trochę overkill? Na pewno da to lepszy podział odpowiedzialności i łatwość testowania.